readme

```diff
@@ -4,7 +4,7 @@ Before running  cdi 2.0 tck you must do the following:
   - Set glassFishHome accordingly
 - Add shared library
   - Copy shared library cdi-tck-ext-lib-2.0.0.Final.jar to $GF_HOME/domains/domain1/lib/applibs
-    This can be found in https://repository.jboss.org/nexus/content/repositories/public/org/jboss/cdi/tck/cdi-tck-ext-lib/2.0.0.Final
+    This can be found in http://central.maven.org/maven2/org/jboss/cdi/tck/cdi-tck-ext-lib/2.0.0.Final/
```

Comment:
Latest is actually 2.0.5.SP1 -> https://search.maven.org/artifact/org.jboss.cdi.tck/cdi-tck-ext-lib/2.0.5.SP1/jar

Reply:
I am just updating the URL as previous one was broken. Also, i have not tested(or certified GlassFish with) v2.0.5.SP1. I propose to update version later.